libraries/rib-base/src/main/java/com/badoo/ribs/core/plugin/Plugin.kt

```diff
@@ -30,6 +30,16 @@ interface NodeLifecycleAware : Plugin {
     fun onDestroy() {}
 }
 
+interface GlobalNodeLifecycleAware : Plugin {
```

Comment:
Can it be implemented as combination of `NodeAware + NodeLifecycleAware`, so in each callback of `NodeLifecycleAware` you will have `node` property from `init` block?

Reply:
As discussed in in person this won't work for the usecase.